src/main/java/org/osc/sdk/manager/api/ManagerDeviceApi.java

```diff
@@ -26,224 +26,190 @@
+     * Finds the device member given its identifier and the context of the current virtual system.
      *
-     * @param id
-     *            The member device id to locate
-     * @return The member device element
-     * @throws Exception
+     * @param id the identifier of the targeted device member
+     * @return The device member, null if not found
+     * @throws Exception upon failure
      */
     ManagerDeviceMemberElement getDeviceMemberById(String id) throws Exception;
 
     /**
-     * Locate member device by name within the members of current VS device container
+     * Finds the device member given its name and the context of the current virtual system.
      *
-     * @param name
-     *            The member device name to locate
-     * @return
-     *         The member device id. Null if not found. {@link ManagerDeviceMemberElement}
-     * @throws Exception
+     * @param name the name of the targeted device member
+     * @return The device member, null if not found
+     * @throws Exception upon failure
      */
     ManagerDeviceMemberElement findDeviceMemberByName(String name) throws Exception;
 
     /**
-     * Return the list of all member device elements for current VS device container
+     * Retrieves the list of all device members for the current virtual system.
      *
-     * @return
-     *         The list of device member elements. List<{@link ManagerDeviceMemberElement}>
-     * @throws Exception
+     * @return the collection of all device members for the current virtual system
+     * @throws Exception upon failure
      */
     List<? extends ManagerDeviceMemberElement> listDeviceMembers() throws Exception;
 
     /**
-     * Check if a appliance upgrade from version represented by srcSwVersion van be upgraded to another version
-     * represented by destSwVersion.
+     * Checks if an appliance upgrade from a provided source version to a
+     * provided target version is supported.
      *
-     * @param modelType
-     *            The appliance model for which version upgrade is being checked.
-     * @param prevSwVersion
-     *            The software version to upgrade from.
-     * @param newSwVersion
-     *            The software version to upgrade to.
-     * @return
-     *         true if upgrade is supported.
-     * @throws Exception
+     * @param modelType  the appliance model for which version upgrade is being checked
+     * @param prevSwVersion  the software version to upgrade from
+     * @param newSwVersion  the software version to upgrade to
+     * @return true if upgrade is supported
+     * @throws Exception upon failure
      */
     boolean isUpgradeSupported(String modelType, String prevSwVersion, String newSwVersion)
             throws Exception;
 
     /**
-     * Get individual member device configuration from Manager to be persisted in ISC. Returned blob will be downloaded
+     * Gets the configuration for a given device member. The returned blob will be downloaded
```

Comment:
"Retrieves the device member..." to be consistent.

get/find: Can it be as retrieve operation? Multiple places